proposals/dictionary-expressions.md

```diff
@@ -213,62 +272,41 @@ The new rules above represent a breaking change: For types that are a valid conv
+**A key_value_pair_element evaluates its interior expressions in order, left to right. In other words, the key is evaluated before the value.**
 
-1. Provide a special syntactic form for the purpose of supplying data to constructors and create methods.  For example: `[new: (comparer: myComp, capacity: 50), "mads": 21, .. ldmMembers]`
+**For each element in order:**
 
-## Construction
+- **If the element is an expression element, the applicable Add instance or extension method is invoked with the element expression as the argument. (Unlike classic collection initializer behavior, element evaluation and Add calls are not necessarily interleaved.)**
 
-> The elements of a collection expression are evaluated in order, left to right. Each element is evaluated exactly once, and any further references to the elements refer to the results of this initial evaluation.
-> 
-> ```diff
-> + A dictionary_element evaluates its interior expressions in order,
-> + left to right.  In other words, the key is evaluated before the value. 
-> ```
->
-> For each element in order:
->
-> - If the element is an expression element, the applicable Add instance or extension method is invoked with the element expression as the argument. (Unlike classic collection initializer behavior, element evaluation and Add calls are not necessarily interleaved.)
-> 
-> - ```diff
->   + If the target is a dictionary type, then the element must be a
->   + `KeyValuePair<,>`.  The applicable indexer is invoked with the
->   + `.Key` and `.Value` members of that pair.
->   ```
->
-> - If the element is a spread element then one of the following is used:
->   - An applicable GetEnumerator instance or extension method is invoked on the spread element expression and for each item from the enumerator the applicable Add instance or extension method is invoked on the collection instance with the item as the argument. If the enumerator implements IDisposable, then Dispose will be called after enumeration, regardless of exceptions.
-> 
->    - ```diff
->      + If the target is a dictionary-type, the enumerator's element
->      + type must be some `KeyValuePair<,>`, and for each of those
->      + elements the applicable indexer is invoked on the collection
->      + instance with the `.Key` and `.Value` members of that pair.
->      ```
+- **If the target is a dictionary type, then the element must be a `KeyValuePair<,>`. The applicable indexer is invoked with the `.Key` and `.Value` members of that pair.**
+
+- **If the element is a spread element then one of the following is used:**
+    - **An applicable GetEnumerator instance or extension method is invoked on the spread element expression and for each item from the enumerator the applicable Add instance or extension method is invoked on the collection instance with the item as the argument. If the enumerator implements `IDisposable`, then `Dispose` will be called after enumeration, regardless of exceptions.**
```

Comment:
```suggestion
    - **If the target is a collection type, an applicable GetEnumerator instance or extension method is invoked on the spread element expression and for each item from the enumerator the applicable Add instance or extension method is invoked on the collection instance with the item as the argument. If the enumerator implements `IDisposable`, then `Dispose` will be called after enumeration, regardless of exceptions.**
```

Reply:
Same concern about Add vs indexer

Reply:
This bullet is for *collection type*, the next bullet is for *dictionary type*.